src/components/Output.jsx

```diff
@@ -59,7 +59,7 @@ class Output extends React.Component {
 
   render() {
     return (
-      <div className="environment__column output">
+      <div className={`environment__column output${this.props.isHidden}`}>
```

Comment:
Definitely like the overall approach here—main thing I’d like to see changed is that in [BEM](https://en.bem.info/methodology/naming-convention/), we always keep the top-level component name class (in this case `output`) and then *add* modifier classes (e.g. `output_hidden`). So, in the hidden case, the classnames of this element should be `environment__column output output_hidden`.

You will likely find the `classnames` helper quite helpful for this.